packages/components/src/AppBar/AppBar.stories.tsx

```diff
@@ -9,17 +9,10 @@
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
 import React from 'react'
-import { ComponentMeta, ComponentStory } from '@storybook/react'
```

Comment:
The _Component_Meta and _Component_Story are actually deprecated but not necessary to fix in this PR.